infra/base-images/base-runner/collect_dft

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash -u
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+cd $OUT
+
+if (( $# > 0 )); then
+  FUZZ_TARGETS="$@"
+else
+  FUZZ_TARGETS="$(find . -maxdepth 1 -type f -executable -printf '%P\n')"
+fi
+
+# Timeout for running a single fuzz target.
+TIMEOUT=1h
+
+# Number of CPUs available, this is needed for running targets in parallel.
+NPROC=$(nproc)
+
+function run_one_target {
+  local target=$1
+  local corpus="/corpus/${target}"
+  local traces="$OUT/${target}_dft"
+  local log="$OUT/${target}_dft.log"
```

Comment:
any reason why log is written to the $OUT apart from debugging?

Reply:
It's pretty large (there is a line printed for every 16 bytes of an input), that's why keeping the logs in separate files. `$OUT` is handy for debugging, right, on CF I think it'll go to a GCS bucket. Added a comment.